src/main/runner/summary.ts

```diff
@@ -1,7 +1,16 @@
-export function handleSummary(data) {
-  const checks = []
+import type { Check } from '@/schemas/k6'
 
-  function traverseGroup(group) {
+interface Group {
+  checks: Check[]
+  groups: Group[]
+}
+
+export function handleSummary(data: {
+  root_group: { checks: Check[]; groups: Group[] }
+}) {
+  const checks: Check[] = []
+
+  function traverseGroup(group: Group) {
```

Comment:
It won't let me comment on unchanged lines, but below the `traverseGroup` function:

```ts
data.root_group.checks.forEach((check) => {
  checks.push(check)
})

data.root_group.groups.forEach((group) => {
  traverseGroup(group)
})

// =>

traverseGroup(data.root_group)
```